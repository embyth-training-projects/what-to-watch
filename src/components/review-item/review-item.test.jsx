import React from "react";
import renderer from "react-test-renderer";

import ReviewItem from "./review-item";

const reviewMock = {
  author: `Amanda Greever`,
  rating: 8.0,
  date: `2019-06-08T14:13:56.569Z`,
  content: `I didn't find it amusing, and while I can appreciate the creativity, it's an hour and 40 minutes I wish I could take back.`,
  id: 4563456345,
};

it(`ReviewItem should render correctly`, () => {
  const tree = renderer
    .create(<ReviewItem
      review={reviewMock}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
