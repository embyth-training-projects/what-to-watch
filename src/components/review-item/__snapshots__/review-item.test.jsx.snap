// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReviewItem should render correctly 1`] = `
<div
  className="review"
>
  <blockquote
    className="review__quote"
  >
    <p
      className="review__text"
    >
      I didn't find it amusing, and while I can appreciate the creativity, it's an hour and 40 minutes I wish I could take back.
    </p>
    <footer
      className="review__details"
    >
      <cite
        className="review__author"
      >
        Amanda Greever
      </cite>
      <time
        className="review__date"
        dateTime="2019-06-08T14:13:56.569Z"
      >
        June 8, 2019
      </time>
    </footer>
  </blockquote>
  <div
    className="review__rating"
  >
    8,0
  </div>
</div>
`;
